src/python/grpcio/grpc/_channel.py

```diff
@@ -827,8 +822,8 @@ def _poll_connectivity(state, channel, initial_try_to_connect):
     completion_queue = cygrpc.CompletionQueue()
     while True:
         channel.watch_connectivity_state(connectivity,
-                                         cygrpc.Timespec(time.time() + 0.2),
-                                         completion_queue, None)
+                                         time.time() + 0.2, completion_queue,
```

Comment:
Should this 0.2 be a named constant?

Reply:
[No, it should be eliminated entirely.](https://github.com/grpc/grpc/issues/3064)